cpp/src/IceSSL/OpenSSLTransceiverI.cpp

```diff
@@ -72,6 +73,101 @@ IceSSL_opensslVerifyCallback(int ok, X509_STORE_CTX* ctx)
 
 }
 
+namespace
+{
+
+TrustError trustStatusToTrustError(long status)
+{
+    switch (status)
```

Comment:
It would be nice if this switch statement had the same order for return values as the order in `TrustError`.

Reply:
Fixed